modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+[id="virt-attaching-vm-secondary-network-cli_{context}"]
+= Attaching a virtual machine to a secondary network in the CLI
+
+Attach a virtual machine to a secondary network by adding a bridge to the `spec.template.spec.domain.devices.interfaces` field and specifying the network attachment definition to the `spec.template.spec.networks` field in the virtual machine configuration.
+
+This procedure uses a YAML file to demonstrate editing the configuration and applying the updated file to the cluster. You can alternatively use the `oc edit <object> <name>` command to edit an existing virtual machine.
+
+If you edit a running virtual machine, you must restart the virtual machine for the changes to take effect. Alternatively, shut down the virtual machine before editing the configuration.
+
+. Create or edit a new virtual machine configuration that you want to connect to the bridge network, for example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: VirtualMachine
+metadata:
+    name: example-vm
+spec:
+  template:
+    spec:
+      domain:
+        devices:
+          interfaces:
+            - masquerade: {}
+              name: default
+            - bridge: {}
+              name: bridge-net <1>
+...
+      networks:
+        - name: default
+          pod: {}
+        - name: bridge-net <1>
+          multus:
+            networkName: <a-bridge-network> <2>
+...
+----
+<1> The name of the bridge interface. This value must match the value of the `spec.networks.name` field.
```

Comment:
a. `spec.networks` is the location of this entry in a VMI yaml. You probably meant `spec.template.spec.networks`.
b. `(spec.template.)spec.networks` is a list, so `spec.networks.name` can't refer to an authentic field. I would go for something like 
"This value must match the value of the `name` field in the corresponding `spec.template.spec.networks` entry."